src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/commands.py

```diff
@@ -102,6 +102,8 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
         g.command('list', 'acr_repository_list')
         g.command('show-tags', 'acr_repository_show_tags')
         g.command('show-manifests', 'acr_repository_show_manifests')
+        g.command('show-attributes', 'acr_repository_show_attributes')
+        g.command('update-attributes', 'acr_repository_update_attributes')
```

Comment:
Why are these not simply `show` and `update`, especially given that the command groups is conspicuously missing these commands.